src/Provider/ApiServiceProvider.php

```diff
@@ -65,7 +65,7 @@ public function register()
             'Dingo\Api\Console\Command\Docs',
         ]);
 
-        if (class_exists('Illuminate\Foundation\Application', false)) {
+        if (class_exists('Illuminate\Contracts\Foundation\Application', false)) {
```

Comment:
@crynobone I'm not sure I follow you. `laravel/lumen` would load `illuminate/contracts` ? False. An application does not "load a namespace". Lumen implements the base contract `Illuminate\Contracts\Container\Container`. Laravel however, implements `Illuminate\Contracts\Foundation\Application`. That is why the author of this plugin is checking to see if that class exists (meaning Laravel is installed).

I'm not checking `class_exists`, the author of this plugin is checking `class_exists`. This is simply a (working) pull request to get the package fixed and working properly. You're only helping detract from the push this pull request needs to help myself and others get this package implemented!